src/cryptonote_core/blockchain.cpp

```diff
@@ -2514,32 +2504,8 @@ bool Blockchain::check_tx_inputs(transaction& tx, tx_verification_context &tvc,
   results.resize(tx.vin.size(), 0);
 
   int threads = tools::get_max_concurrency();
```

Comment:
I would change this to be retrieved from the `threadpool`, so that if the implementation changes how it calculates the number of threads for some reason it remains correct.

Reply:
Ok.